<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Characters DB</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Fichier">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
    <addaction name="action_new_db"/>
    <addaction name="action_ope_db"/>
   </widget>
   <widget class="QMenu" name="menu_Personnages">
    <property name="title">
     <string>&amp;Personnages</string>
    </property>
    <addaction name="action_char_list"/>
    <addaction name="action_new_char"/>
   </widget>
   <addaction name="menu_Fichier"/>
   <addaction name="menu_Personnages"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="ListCharWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QListWidget" name="charactere_list"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_new_db">
   <property name="text">
    <string>&amp;Nouvelle base de données</string>
   </property>
  </action>
  <action name="action_ope_db">
   <property name="text">
    <string>Ouvrir Base de données</string>
   </property>
  </action>
  <action name="action_char_list">
   <property name="text">
    <string>&amp;Liste de personnages</string>
   </property>
  </action>
  <action name="action_new_char">
   <property name="text">
    <string>&amp;Nouveau personnage</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
